packages/react-composites/src/composites/CallWithChatComposite/components/MoreDrawer.tsx

```diff
@@ -142,8 +143,9 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
         // But this is a known issue in our state.
         onSelectMicrophone(selected as AudioDeviceInfo);
       }
+      props.onLightDismiss();
     },
-    [microphones, onSelectMicrophone]
+    [microphones, onSelectMicrophone, props]
```

Comment:
(nit: same here on avoiding props in the dep array)

Reply:
^^ I second this as well

Reply:
updated :)